test/commands/test_remote.rb

```diff
@@ -116,4 +116,42 @@ def test_remote_trim_with_tolerate
       assert_equal(1, remotes.all.count)
     end
   end
+
+  # @todo #329:30min Verify that the nodes that are being selected are
+  #  really the strongest ones. The strongest nodes are the ones with
+  #  the highest score.
+  def test_select_selects_the_strongest_nodes
```

Comment:
@botanicus maybe `test_strongest_nodes_are_selected`

Reply:
What I’m trying to do here is to (sort of) namespace the subcommands. The `remote` command has many subcommands and currently we’re testing them in one Minitest class.

With this convention (`test_subcommandname_testdesc`) one can see which tests test which subcommand. I’m not saying it’s the best approach though, rather an ad-how one.

I think we need a better solution than that though, so I reported it here #360.

Reply:
@botanicus got you. In RSpec you could say `describe '#select'`.

Reply:
@ledestin exactly. I missed the `#`. Updated now, cheers!